File: 900/CF2209B_Array.java

import java.util.*;

public class CF2209B_Array {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int t = sc.nextInt();

        while (t-- > 0) {
            int n = sc.nextInt();

            int[] a = new int[n];

            for (int i = 0; i < n; i++) {
                a[i] = sc.nextInt();
            }

            int[] ans = new int[n];

            for (int i = 0; i < n; i++) {
                int smaller = 0;
                int greater = 0;

                for (int j = i + 1; j < n; j++) {
                    if (a[j] < a[i]) {
                        smaller++;
                    } else if (a[j] > a[i]) {
                        greater++;
                    }
                }

                ans[i] = Math.max(smaller, greater);
            }

            for (int i = 0; i < n; i++) {
                System.out.print(ans[i] + " ");
            }

            System.out.println();
        }

        sc.close();
    }
}

File: 900/CF2209B_Array.py
t = int(input())

for _ in range(t):
    n = int(input())
    a = list(map(int, input().split()))

    ans = []

    for i in range(n):
        smaller = 0
        greater = 0

        for j in range(i + 1, n):
            if a[j] < a[i]:
                smaller += 1
            elif a[j] > a[i]:
                greater += 1

        ans.append(max(smaller, greater))

    print(*ans)